### PR TITLE
Create multiple istio IPs

### PR DESCRIPTION
**Pull request by syedrakib**
Commit-by-Commit:
- Allow creating multiple Istio IPs
- Remove dependency on var.create_istio_components

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -47,14 +47,14 @@ variable "ingress_ip_names" {
   default     = []
 }
 
-variable "istio_ip_name" {
-  description = "An arbitrary name to identify the Istio IP that will be generated for the k8s cluster if \"var.create_istio_components\" is set to \"true\"."
-  type        = string
-  default     = "istio-ip"
+variable "istio_ip_names" {
```
Comment (dibosh) [inline variables.tf:50]: let's not say istio_ip_names, call it ingress_ip_names. these ips are anyhow to be used with all kinds of ingress. istio gateway is just ingress.

**Review by dibosh — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (syedrakib) [reply variables.tf:50]: We already have `variable "ingress_ip_names"` defined. And that's what we tried to use originally for setting up Istio.

Later, we figured the general ingress IPs require to be **Global** External IPs while the Istio IPs require to be **Regional** External IPs.

Hence, the distinction.

Comment (dibosh) [reply variables.tf:50]: I see.

**Review by dibosh — APPROVED** [review]
LGTM

**Pull request merged**